Build/CMakeLists.txt

```diff
@@ -92,7 +91,9 @@ if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Windows" OR "${CMAKE_SYSTEM_NAME}" STREQUAL
 		set(CMAKE_STATIC_LINKER_FLAGS_RELEASE "/LTCG")
 	elseif ("${CMAKE_CXX_COMPILER_ID}" STREQUAL "Clang")
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} /showFilenames")
-		if (USE_AVX2)
+		if (USE_AVX512)
+			set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -mavx512f -mavx512vl -mavx2 -mbmi -mpopcnt -mlzcnt -mf16c -mfma")
```

Comment:
I removed -mfma yesterday because it breaks cross platform determinism, by default it is added below again (so it used to be 2x on the commandline). Can you remove it?